### PR TITLE
Add variations typos of 'trick'

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -28695,6 +28695,10 @@ triggeres->triggers
 triggerred->triggered
 triggger->trigger
 triguered->triggered
+trik->trick
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28698]: Suggest trike too

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -28695,6 +28695,10 @@ triggeres->triggers
...
+triked->tricked
+triks->tricks
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28700]: Also trikes

**Review by peternewman — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 8c45e8b

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 8c45e8b

**Comment by luzpaz** [comment]
Added requested revisions + added `trikery->trickery`

Please merge

**Review by peternewman — APPROVED** [review]

**Pull request merged**